editor/editor_node.cpp

```diff
@@ -7824,6 +7824,8 @@ EditorNode::EditorNode() {
 
 	log = memnew(EditorLog);
 	Button *output_button = add_bottom_panel_item(TTR("Output"), log);
+	// Add separation for the warning/error icon that is displayed later.
+	output_button->add_theme_constant_override("h_separation", 6 * EDSCALE);
```

Comment:
We should move this to the editor theme and reuse it between here and the debugger.

Reply:
Done.